orangecontrib/spectroscopy/widgets/owhyper.py

```diff
@@ -1116,6 +1350,24 @@ class Outputs(SelectionOutputsMixin.Outputs):
     rgb_green_value = ContextSetting(None)
     rgb_blue_value = ContextSetting(None)
 
+    show_vector_plot = Setting(False)
+    vector_angle = ContextSetting(None)
+    vector_magnitude = ContextSetting(None)
+    vector_colour_index = Setting(0)
+    vcol_byval_index = Setting(0)
+    vcol_byval_feat = ContextSetting(None)
+    vector_scale = Setting(1)
+    vector_width = Setting(1)
+    vector_opacity = Setting(255)
+    v_bin = Setting(0)
+
+    a = None
```

Comment:
Most of this vars should be saved in the image_plot, as they only concern the image